packages/email-service/cronJobs/run-campaigns.ts

```diff
@@ -161,15 +169,42 @@ async function getSendMailOptions(
   recipient: any,
   emailAccount: any
 ) {
+
+  // determine which email has to be sent and whether to send
+
+  // count how many emails have already been sent to the user to determine which email to send next in the sequence
```

Comment:
what if the drip campaign changes? or if we remove an email from the drip campaign sequence?

Reply:
removing an email from the drip campaign will affect the order of the sequence, 
Iv push a solution for this where if an email template gets deleted from the drip campaign, all records of that email template getting sent are deleted, that should fix the counting issue